physionet-django/project/modelcomponents/access.py

```diff
@@ -167,6 +168,35 @@ class Meta:
         default_permissions = ()
 
 
+class DataSource(models.Model):
```

Comment:
I think we'll need some proper validations on this model (require URI for certain mechanisms, require email for certain mechanisms etc.)